production/ksonnet/loki/overrides.libsonnet

```diff
@@ -25,14 +25,6 @@ local k = import 'ksonnet-util/kausal.libsonnet';
       //   log_push_request_streams: true,
       // },
     },
-
-    defaultRuntimeConfigs: {
-      // override the default runtime configs. Useful to change a behavior globally across multiple tenants. See pkg/loki/runtime_config.go
-      //
-      // log_stream_creation: true,
-      // log_push_request: true,
-      // log_push_request_streams: true,
-    },
   },
   local configMap = k.core.v1.configMap,
 
```

Comment:
i think you'll have to remove this line too
```diff
  overrides_config:
    configMap.new($._config.overrides_configmap_name) +
    configMap.withData({
      'overrides.yaml': k.util.manifestYaml(
        {
          overrides: $._config.overrides,
          configs: $._config.runtimeConfigs,
-         default: $._config.defaultRuntimeConfigs,
        }
        + (if std.length($._config.multi_kv_config) > 0 then { multi_kv_config: $._config.multi_kv_config } else {}),
      ),
    }),
```

Reply:
ah good catch, ty